lib/jsdom/living/nodes/HTMLAnchorElement-impl.js

```diff
@@ -9,8 +9,10 @@ class HTMLAnchorElementImpl extends HTMLElementImpl {
     super(globalObject, args, privateData);
 
     this._htmlHyperlinkElementUtilsSetup();
+  }
 
-    this._hasActivationBehavior = true;
+  get _hasActivationBehavior() {
+    return this.hasAttributeNS(null, "href");
```

Comment:
You said

> I misunderstood before, not all elements has activation behavior are interactive content. like `HTMLAreaElement`

Should these changes be reverted then?

Reply:
If this revert, activation behavior of `<a></a>` will be used. This may not the desired result. https://github.com/jsdom/jsdom/blob/master/lib/jsdom/living/events/EventTarget-impl.js#L121-L123